pom.xml

```diff
@@ -5,7 +5,7 @@
     <!-- groupId, artifactId, these namespaces should not be changed -->
     <groupId>ai.rev</groupId>
     <artifactId>revai-java-sdk</artifactId>
-    <version>2.4.0</version>
+    <version>2.4.1</version>
```

Comment:
I think [this](https://semver.org/#summary) is relevant to the change

not sure whether it was discussed.
as a fact, this is a breaking change so it should be `3.0.0`.
but technically we have not announced/documented this feature yet.